CLR/Graphics/Graphics.cpp

```diff
@@ -195,7 +195,7 @@ HRESULT CLR_GFX_Bitmap::CreateInstance( CLR_RT_HeapBlock& ref, const CLR_UINT8*
         /* When loading a Windows BMP, GIF, or JPEG file, it is converted in-place to the native BPP.
          * When loading a compressed TinyCLR Bitmap from a resource file, two bitmaps are needed to decompress, then convert.
          * This fragments the heap and wastes space until the next garbage collection is done.
-         * When using the SimpleHeap, there is no relocation, so decompressing into a temp bitmap into the simpleheap wastes
+         * When using the heap, there is no relocation, so decompressing into a temp bitmap into the heap wastes

```

Comment:
I'm not following you here... A platform creator can make the standard heap as large as he wants to hold these graph and whatever stuff it's needed. I don't see why this comment is misleading or not providing enough explanations for an informed decision.